### ISSUE TITLE
Accessibility: Incomplete view of publishing options with Large Text option enabled

### ISSUE THREAD

**Issue by reginabally** [wordpress-mobile/WordPress-iOS#15707]
It was reported in 3626547-zen that the bottom part of the publishing options in the app wasn't visible and not scrollable.

### Expected behavior

I would expect to be able to scroll down to see the rest of the publishing options if the text wasn't able to fit into the screen.

### Actual behavior

I can only see the top section of the publishing options but not all. And I wasn't able to scroll down to see the rest of the options.

### Steps to reproduce the behavior

1. Go to iOS Settings > Accessibility > Display & Text Size and enable Large Text.
2. Change the text size to larger text size.
3. Go to the WPiOS app > My Site > Blog Posts > Drafts and select a draft post.
4. Tap the three dots and choose Publish.
5. See that the bottom part of the publishing options are not visible and not being able to scroll down to see the rest of the options.

![IMG_8344D430742D-1](https://user-images.githubusercontent.com/40906847/105789811-0d824c00-5fbe-11eb-8eaa-61c91c8bd8cb.jpeg)

**Comment by leandroalonso** [comment]
JFYI: we fixed the scrolling issue in #15716 

However, we'd like to display the nudges with the correct height right away, that's why we're keeping this issue open and will iterate again in the future.

**Comment by rezzap** [comment]
There's a report of a similar issue here in the iPhone Pro Max, though it doesn't sound like they have accessibility settings enabled.

>When creating a post on the app for an iPhone Pro Max, the post/schedule bar at the bottom of the page (pink rectangle) only partially appears on the screen and you cannot move it up further to press. The workaround is to tip the phone on the side and then back, this displays nearly all of that button.

I'm asking the user for a screenshot just to be sure it's related and letting them know that we're working on a better fix for the next version. 

3704529-zen

**Comment by rossanafmenezes** [comment]
> There's a report of a similar issue here in the iPhone Pro Max, though it doesn't sound like they have accessibility settings enabled.
> 
> > When creating a post on the app for an iPhone Pro Max, the post/schedule bar at the bottom of the page (pink rectangle) only partially appears on the screen and you cannot move it up further to press. The workaround is to tip the phone on the side and then back, this displays nearly all of that button.
> 
> I'm asking the user for a screenshot just to be sure it's related and letting them know that we're working on a better fix for the next version.
> 
> 3704529-zen

User sent screenshots (in the ticket). Looks like the same issue

**Comment by leandroalonso** [comment]
@rezzap @rossanafmenezes can you please let the user know that the next version (that is going to be released next week) has the fix to this issue?

**Comment by diegoreymendez** [comment]
@leandroalonso - I'm checking out this issue as part of the NUX Larger Text / Internationalization board.

I was wondering if we could close this issue and open a new one that details a bit more specifically the work that remains to be done.  By doing that I could either help tackling the new issue, or move out of scope (if we deem it lower priority).

**Comment by leandroalonso** [comment]
@diegoreymendez I believe that the original issue was completely solved in #15716. The user is capable to scroll the bottom sheet up if the content doesn't fully appear.

I spotted a different issue though: the combination of larger font + VoiceOver makes it impossible for the user to scroll the bottom sheet up (using the 3 fingers gesture). We might want to close this one and create another issue specifically for this one.

Thoughts, @emilylaguna?

**Comment by emilylaguna** [comment]
Sounds good to me!

**Comment by leandroalonso** [comment]
@diegoreymendez I've created this new issue: https://github.com/wordpress-mobile/WordPress-iOS/issues/16460

Thanks for the ping!
